app/Nova/Actions/ActivateUitpasIntegration.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova\Actions;
+
+use App\Domain\Integrations\Models\IntegrationModel;
+use App\Domain\Integrations\Organizer;
+use App\Domain\Integrations\Repositories\IntegrationRepository;
+use App\Domain\Integrations\Repositories\OrganizerRepository;
+use App\Domain\Organizations\Models\OrganizationModel;
+use Illuminate\Bus\Queueable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Support\Collection;
+use Laravel\Nova\Actions\Action;
+use Laravel\Nova\Actions\ActionResponse;
+use Laravel\Nova\Fields\ActionFields;
+use Laravel\Nova\Fields\Select;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Http\Requests\NovaRequest;
+use Ramsey\Uuid\Uuid;
+
+final class ActivateUitpasIntegration extends Action
+{
+    use InteractsWithQueue;
+    use Queueable;
+
+    public function __construct(
+        private readonly IntegrationRepository $integrationRepository,
+        private readonly OrganizerRepository $organizerRepository
+    ) {
+    }
+
+    public function handle(ActionFields $fields, Collection $integrations): ActionResponse
+    {
+        /** @var IntegrationModel $integration */
+        $integration = $integrations->first();
```

Comment:
Are we sure we will always have an Integration?
Otherwise I would run a check with an instanceof.
That is also a way to avoid the docblock.

Reply:
Since it's an action on an Integration, there should always be at least 1 Integration.